src/components/ChartMenu/Chart/Chart.tsx

```diff
@@ -0,0 +1,34 @@
+import styles from "@components/ChartMenu/Chart/Chart.module.scss";
+import { Legend } from "@components/ChartMenu/Chart/Legend/Legend";
+import { ChartElement } from "@components/ChartMenu/ChartElement";
+import { DragEvent } from "react";
+import { LinesCanvas } from "@components/ChartMenu/Chart/LinesCanvas/LinesCanvas";
+type Props = {
+  chartElement: ChartElement;
+  handleDropOnChart: (chartId: number, measurementName: string) => void;
+};
+
+export const Chart = ({ chartElement, handleDropOnChart }: Props) => {
+  function handleDrop(ev: DragEvent<HTMLDivElement>) {
+    ev.stopPropagation();
+    handleDropOnChart(chartElement.id, ev.dataTransfer.getData("text/plain"));
+  }
```

Comment:
I would like you to explain to me how this works in concrete terms at future meetings